backend/infrahub/graphql/mutations/main.py

```diff
@@ -274,7 +276,9 @@ async def mutate_update_object(
     ) -> Node:
         context: GraphqlContext = info.context
         component_registry = get_component_registry()
-        node_constraint_runner = await component_registry.get_component(NodeConstraintRunner, db=db, branch=branch)
+        node_constraint_runner = await component_registry.get_component(
+            NodeConstraintRunner, db=db.start_session(read_only=True), branch=branch
```

Comment:
Since we are using a transaction for the mutation we  need to explicitly start a new db session in READ mode to run the NodeConstraint queries.